app/.env.example

```diff
@@ -0,0 +1 @@
+GOOGLE_CLIENT_ID=''
```

Comment:
shouldn't this file have other keys as well?

Reply:
In app, we only require this key

Reply:
What other keys are we expecting here?

Reply:
ok. I guess we will be adding sentry pretty soon.